docs/demo/scripts/custom_reservoir_handler.py

```diff
@@ -0,0 +1,25 @@
+from wsimod.extensions import register_node_patch
+
+
+@register_node_patch("my_reservoir", "pull_set_handler", item="FWTW")
+def custom_pulls_fwtw(self, vqip, *args, **kwargs):
+    """A custom handler function."""
```

Comment:
What about a little bit more descriptive explanations? I don't think it matters, but it just looks odd to have lots of functions with exactly the same docstring.